pkgs/build-support/build-sandbox/src/setup.c

```diff
@@ -668,6 +671,7 @@ static bool setup_static_etc(struct query_state *qs)
     char dest[PATH_MAX];
     ssize_t destlen;
 
+    // TODO: what does this do? Why is dest needed?
     if ((destlen = readlink("/etc/static", dest, PATH_MAX)) == -1)
```

Comment:
`dest` is the buffer where the result of the symlink resolve is stored.

Reply:
Added a docstring to the subroutine.